paper-ripple.html

```diff
@@ -476,6 +493,13 @@
         return false;
       },
 
+      simulatedRipple: function() {
+        this.mousedownAction();
+        this.async(function() {
+          this.mouseupAction();
+        }, 1);
```

Comment:
I have filed polymer/polymer#1305 and will make a comment linking there to explain usage of `this.async(..., 1);` where appropriate.